src/geometry/quaternion.rs

```diff
@@ -506,6 +527,22 @@ impl<N: Real> Quaternion<N> {
     pub fn normalize_mut(&mut self) -> N {
         self.coords.normalize_mut()
     }
+
+    /// Calculates quaternionic sin
+    #[inline]
+    pub fn cos(&self) -> Self {
+        let z = self.imag().magnitude();
+        let w = -self.w.sin() * sinhc(z);
+        Self::from_parts(self.w.cos() * z.cosh(), self.imag() * w)
+    }
+
+    /// Calculates quaternionic sin
```

Comment:
```suggestion
    /// Calculates the quaternionic sinus.
```

Reply:
I think "sine" and "cosine" are the most common spellings of these in English.